plugins/ltac/g_rewrite.mlg

```diff
@@ -279,8 +279,7 @@ VERNAC COMMAND EXTEND AddSetoid1 CLASSIFIED AS SIDEFF
          add_setoid atts binders a aeq t n;
      }
   | #[ atts = rewrite_attributes; ] [ "Add" "Morphism" constr(m) ":" ident(n) ]
-    (* This command may or may not open a goal *)
-    => { VtUnknown, VtNow }
+    => { VtStartProof(GuaranteesOpacity, [n]), VtLater }
```

Comment:
I think this is not quite correct, program_mode is included in rewrite_attributes and this command just forwards to the instance command so it has the same behaviour.
The current coqpp setup doesn't make the attributes available to the classify expression so we need to modify it to implement the correct thing. Otherwise we could put user_err "program mode not supported" in add_morphism_infer and leave the attribute issue for later (assuming nobody relies on program mode Add Morphism).

Reply:
Never mind, it passes a partial term so never opens a proof.